### PR TITLE
Release 6.0.0

### PR DESCRIPTION
**Pull request by rlepinski**
Release 6.0.0. 
- Updated Urban Airship iOS SDK to 10.0.3
- Updated Urban Airship Android SDK to 9.5.6
- Removed ability to set/get an alias. Use named user instead.

Tested in iOS and Android. Able to receive push in both.

### CODE REVIEW

File: airship.properties [hunk 1]
```diff
@@ -19,7 +19,7 @@ androidTargetSdkVersion = 27
 # Android min SDK version
 androidMinSdkVersion = 16
 
-# Firebase Core
+# Firebase C
```
Comment (BrianBatchelder) [inline airship.properties]: Unintentional?

File: unity-plugin/build.gradle [hunk 2]
```diff
@@ -1,7 +1,7 @@
 buildscript {
     repositories {
-        jcenter()
         google()
+        jcenter()
```
Comment (crow) [inline unity-plugin/build.gradle]: v important

**Comment by crow** [comment]
🍔

**Comment by marc-scig** [comment]
👍

**Pull request merged**